poetry.lock

```diff
@@ -1151,14 +1152,14 @@ doc = ["Sphinx", "sphinx-rtd-theme"]
 
 [[package]]
 name = "setuptools"
-version = "67.0.0"
```

Comment:
Similar question here, why don't we stay upgraded to the current version?
I also see the category change from dev to main - that probably means setuptools is promoted from a dev dependency to a main dependency, I'd prefer to update the constraint in pyproject to have it at the same version as currently in the lock file.

Reply:
reverted the change to min instead of `^`